### PR TITLE
GWTCHOS-46 : select search item in mobile clear search results

### PR DESCRIPTION
**Pull request by jasonlemay**

### CODE REVIEW

**Comment by jasonlemay** [comment]
@meriouma like that?

**Comment by meriouma** [comment]
LGTM

File: plugin/src/main/java/com/arcbees/chosen/client/AbstractMobileChosenImpl.java [hunk 1]
```diff
@@ -150,6 +152,11 @@ protected boolean searchResultsMouseOut(Event e) {
         return super.searchResultsMouseOut(e);
     }
 
+//    @Override
```
Comment (christiangoudreau) [inline plugin/src/main/java/com/arcbees/chosen/client/AbstractMobileChosenImpl.java]: Has this commented code been merged?

Comment (jasonlemay) [inline plugin/src/main/java/com/arcbees/chosen/client/AbstractMobileChosenImpl.java]: @christiangoudreau Yes it's merged but I'm working on Chosen right now. @meriouma can we remove this function or it is required and was just commented as a test?

Comment (olafleur) [inline plugin/src/main/java/com/arcbees/chosen/client/AbstractMobileChosenImpl.java]: Not in master.

Comment (meriouma) [inline plugin/src/main/java/com/arcbees/chosen/client/AbstractMobileChosenImpl.java]: Yes you can remove it

**Pull request merged**